### PR TITLE
Initial draft of Introducing Nex blog post

### PR DESCRIPTION
**Pull request by autodidaddict**

### CODE REVIEW

File: content/blog/introducing_nex.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+* **Functions** - These are short-lived, on-demand units of compute that are triggered by some stimulus and return some value. You see this category everywhere, including CloudFlare, Amazon, Azure, Google, and more.
+* **Services** - Long-running units of compute that start, provide some service, and don't stop until explicitly told to stop via control plane. Common types of services are applications deployed directly to virtual machines or containers run in Kubernetes or other runtimes.
+
+With Nex, you can deploy _zero dependency_ **JavaScript** and **WebAssembly** functions as well as native, 64-bit Linux statically compiled services. All of these deployments can be monitored, interrogated, and controlled through the same unified interface. You can even deploy and run OCI containers if you absolutely must.
+
+If you're at all curious about Nex, check out our **[Getting Started](https://docs.nats.io/using-nats/nex)** guide. While it's far from complete and has the rough edges you might expect of a tech preview, we can't wait to start the feedback loop and get it into the hands of real NATS users and see the amazing things people build with it.
```
Comment (bruth) [inline content/blog/introducing_nex.md:27]:
```suggestion
If you're at all curious about Nex, check out the [unveiling announcement and live demo](https://www.youtube.com/watch?v=EfxtiKMnoyQ&list=PLgqCaaYodvKZfOFK3MMk5YnSO0cNj-QuQ&index=9) from RethinkConn 2024 to see it in action. After that, check out the **[getting Started](https://docs.nats.io/using-nats/nex)** guide. While it's far from complete and has the rough edges you might expect of a tech preview, we can't wait to start the feedback loop and get it into the hands of real NATS users and see the amazing things people build with it.
```

Could be reworded, but wanted to slip in the video.

**Review by bruth — COMMENTED** [review]
LGTM! Will let @gcolliso and others do the final review and approval.

**Review by derekcollison — COMMENTED** [review]
We should create a nex channel in the NATS.io slack for the OSS community. Might be better to link to that one vs Synadia which is invite only etc.

Byron WDYT?

Otherwise looks good!

**Comment by autodidaddict** [comment]
> We should create a nex channel in the NATS.io slack for the OSS community. Might be better to link to that one vs Synadia which is invite only etc.
> 
> Byron WDYT?
> 
> Otherwise looks good!

That channel is already there.

**Comment by derekcollison** [comment]
Should this point to that vs synadia one?

**Comment by autodidaddict** [comment]
It does point to the nats slack and not the synadia one

**Comment by derekcollison** [comment]
Yep my bad, you are correct, read it wrong. Sorry for the unnecessary diversion.

**Comment by autodidaddict** [comment]
> Yep my bad, you are correct, read it wrong. Sorry for the unnecessary diversion.

Wherever we are is where we're meant to be 😀

**Review by gcolliso — APPROVED** [review]
lgtm

**Pull request merged**